homeassistant/components/airvisual/__init__.py

```diff
@@ -358,11 +359,15 @@ async def async_reload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
 class AirVisualEntity(CoordinatorEntity):
     """Define a generic AirVisual entity."""
 
-    def __init__(self, coordinator: DataUpdateCoordinator) -> None:
+    def __init__(
+        self, coordinator: DataUpdateCoordinator, description: EntityDescription
+    ) -> None:
         """Initialize."""
         super().__init__(coordinator)
 
         self._attr_extra_state_attributes = {ATTR_ATTRIBUTION: DEFAULT_ATTRIBUTION}
+        self._kind = description.key
```

Comment:
replace the use of `self._kind` in the code with `self.entity_description.key`, removing the need for this additional variable.

```suggestion
```